src/main/java/com/minecolonies/coremod/network/messages/SaveScanMessage.java

```diff
@@ -41,7 +49,33 @@ public SaveScanMessage(final NBTTagCompound nbttagcompound, final String storeAt
     @Override
     public void fromBytes(@NotNull final ByteBuf buf)
     {
-        nbttagcompound = ByteBufUtils.readTag(buf);
+        PacketBuffer pb = new PacketBuffer(buf);
+        ByteBufInputStream stream = new ByteBufInputStream(pb);
+
+        try
+        {
+            nbttagcompound = CompressedStreamTools.read(stream, new NBTSizeTracker(200097152L));
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Assign this magic number 200097152L to a well-named constant, and use the constant instead. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS109)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Add underscores to this numeric value for readability [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS2148)